scripts/generate_dump

```diff
@@ -758,11 +758,28 @@ enable_logrotate() {
 #  None
 ###############################################################################
 collect_mellanox() {
-    local sai_dump_filename="/tmp/sai_sdk_dump_$(date +"%m_%d_%Y_%I_%M_%p")"
+    local sai_dump_folder="/tmp/saisdkdump"
+    local sai_dump_filename="${sai_dump_folder}/sai_sdk_dump_$(date +"%m_%d_%Y_%I_%M_%p")"
+
+    ${CMD_PREFIX}docker exec -it syncd mkdir -p $sai_dump_folder
     ${CMD_PREFIX}docker exec -it syncd saisdkdump -f $sai_dump_filename
-    ${CMD_PREFIX}docker exec syncd tar Ccf $(dirname $sai_dump_filename) - $(basename $sai_dump_filename) | tar Cxf /tmp/ -
-    save_file $sai_dump_filename sai_sdk_dump true
 
+    copy_from_docker syncd $sai_dump_folder $sai_dump_folder
+    echo "$sai_dump_folder"
+    for file in `ls $sai_dump_folder`; do
+        save_file ${sai_dump_folder}/${file} sai_sdk_dump true 
+    done
+
+    ${CMD_PREFIX}rm -rf $sai_dump_folder
+    ${CMD_PREFIX}docker exec -it syncd rm -rf $sai_dump_folder
+
+    local mlxtrace_file=`${CMD_PREFIX}docker exec syncd find /tmp -type f -iname "mt*_pci_cr0_mlxtrace.trc"`
+    if [[ $mlxtrace_file ]]; then
+        copy_from_docker syncd $mlxtrace_file $mlxtrace_file
+        save_file $mlxtrace_file mlxtrace false
```

Comment:
I think these commands are redundant now after collecting all logs from sai dump dir.
@DavidZagury please double check it.